src/Features/VisualBasic/Portable/CodeFixes/OverloadBase/OverloadBaseCodeFixProvider.vb

```diff
@@ -30,7 +30,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.OverloadBase
                 Return
             End If
 
-            context.RegisterCodeFix(New AddOverloadsKeywordAction(context.Document, token.Parent), context.Diagnostics)
+            If diagnostic.Descriptor.Id = BC40003 Then
+                context.RegisterCodeFix(New AddOverloadsKeywordAction(context.Document, token.Parent), context.Diagnostics)
+            End If
+            If diagnostic.Descriptor.Id = BC40004 Then
+                context.RegisterCodeFix(New AddShadowsKeywordAction(context.Document, token.Parent), context.Diagnostics)
```

Comment:
as mentioned previously.  These two actions are so similar that they really should just be one parameterizable action.  (or at least have an abstract base action that contains the majority of shared logic).